### DISCUSSION TITLE
The Hunt interface may contain additional log messages to further diagnose the issue.

### DISCUSSION THREAD

**Discussion by parsivalllll** [Security-Onion-Solutions/securityonion#11490] [2.4]
### Version

2.4.0

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

16

### Storage for /

200 GB

### Storage for /nsm

200 GB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Request failed with status code 502
See the Help section of the Security Onion documentation for additional troubleshooting guidance. 
The Hunt interface may contain additional log messages to further diagnose the issue. 

This is the Error message I'm getting to enter the dashboard, Alerts, Cases, and Hunt tabs. All the services are up and running 

from the logs: [2023-10-09T23:59:05,975][WARN ][org.elasticsearch.cluster.routing.allocation.DiskThresholdMonitor] high disk watermark [85%] exceeded on [oCwATom8T4uMYSJBrN5btQ][seseconion][/usr/share/elasticsearch/data] free: 6.1gb[11%], shards will be relocated away from this node; currently relocating away shards totalling [0] bytes; the node is expected to continue to exceed the high disk watermark when these relocations are complete
![2](https://github.com/Security-Onion-Solutions/securityonion/assets/133628068/e1e06739-91b3-4196-bbdb-77cbeee98abe)
![1](https://github.com/Security-Onion-Solutions/securityonion/assets/133628068/0ca489c0-ad47-4cf0-bcca-6d8698d5e96b)

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
16GB RAM does not meet the minimum requirement for Standalone. From https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments:
> In a standalone deployment, the manager components and the sensor components all run on a single box, therefore, your hardware requirements will reflect that. You’ll need at minimum 24GB RAM, 4 CPU cores, and 200GB storage. At the bare minimum of 24GB RAM, you would most likely need swap space to avoid issues.
